csm_web/frontend/src/components/course/SectionCard.tsx

```diff
@@ -90,13 +90,17 @@ export const SectionCard = ({
     const iconWidth = "8em";
     const iconHeight = "8em";
     if (enrollmentSuccessful) {
+      const inlineIconWidth = "1.3em";
+      const inlineIconHeight = "1.3em";
       return (
         <div className="enroll-confirm-modal-contents">
           <CheckCircle height={iconHeight} width={iconWidth} />
           <h3>Successfully enrolled</h3>
-          <ModalCloser>
-            <button className="primary-btn">OK</button>
-          </ModalCloser>
+          <h4>To view and update your profile, click the button below</h4>
+          <Link className="primary-btn" to="/profile">
+            <UserIcon width={inlineIconWidth} height={inlineIconHeight} />
+            Profile
+          </Link>
```

Comment:
This change is the cause of the Cypress test failures; the Cypress tests should change to look for `Profile` instead, and should make sure that the user is redirected as desired.